R/GTM.R

```diff
@@ -0,0 +1,66 @@
+#' GTM
+#' This function carries out gene text mining.
+#' @param genes A character string containing the genes
+#' to perform text mining using PubMed database
+#' @param query_string A character string containing words 
+#' in query to follow the gene of interest. Default is 
+#' "AND cancer AND driver" resulting in a final query of
+#' "Gene AND cancer AND driver". Standard PubMed syntax
+#' can be used in the query. For example Boolean operators
+#' AND, OR, NOT can be applied and tags such as [AU], 
+#' [TITLE/ABSTRACT], [Affiliation] can be used. 
+#' @import easyPubMed
+#' @import tibble
+#' @importFrom magrittr "%>%"
+#' @importFrom purrr map
+#' @importFrom dplyr rename
+#' @return A tibble containing results of text mining where
```

Comment:
I would call this literature search more than text mining - text usually mining involves parsing the text you find and extracting relevant parts from a text/deriving some sort of compressed representation of the information (e.g. extracting some sentences/text out of an article)